content/python/concepts/unittest-module/unittest-module.md

```diff
@@ -0,0 +1,108 @@
+---
+Title: 'Unittest Module'
+Description: 'The unittest module provides features for unit testing Python code.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Automation'
+  - 'Debugging'
+  - 'Development'
+  - 'Modules'
+  - 'Units'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+---
+
+The **`unittest`** [module](https://www.codecademy.com/resources/docs/python/modules) provides features for [unit testing](https://www.codecademy.com/resources/docs/general/unit-testing) Python code. Similar to unit testing frameworks in other languages ,`unittest` allows for the automation of unit tests, sharing of setup and shutdown code for tests, and aggregation of tests into collections.
```

Comment:
Instead of "shutdown" use "tear down" to be more consistent with the function name: `tearDown()`.